tests/nested/manual/muinstaller-real/task.yaml

```diff
@@ -362,3 +249,8 @@ execute: |
   remote.exec snap list "core${version}" > "core${version}".after-boot
   # and check that the versions are the same after boot
   diff -u "core${version}".before-boot "core${version}".after-boot
+
+  if [ "$PARTIAL_GADGET" = true ]; then
+      # make sure that our gpt partition is still there
+      remote.exec "sudo fdisk /dev/vda1 -l" | MATCH "BIOS Boot"
```

Comment:
Is this really working? It does not look like this is showing the partition label, maybe you need to look at `dev/disk/by-partlabel/` for this. And then `sudo fdisk /dev/vda -l` to see start and size (`vda1` will not show you start of the partition).

Reply:
You're right, this was a guess that I never came back to. :) It doesn't work. Your suggestion should work though, thanks.